lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net471</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp2.1;net471</TargetFrameworks>

```

Comment:
Same here

Reply:
Other projects ok, but if not keep both `netcoreapp2.0;netcoreapp2.1;` here then it will not be possible to run tests on different dotnet core versions, isn't it? 
```
 dotnet test -f netcoreapp2.0 -s test.runsettings
 dotnet test -f netcoreapp2.1 -s test.runsettings
```
Or it should be tested only by the latest dotnet?

Reply:
You're right. Maybe I wouldn't test both versions on builds (I would test 2.1), but we could leave these frameworks available for testing.
So leave all of these but PuppeteerSharp, which needs to be NetStandard.